services/accounts/accounts.go

```diff
@@ -49,7 +49,23 @@ func (api *API) SaveAccounts(ctx context.Context, accounts []accounts.Account) e
 }
 
 func (api *API) GetAccounts(ctx context.Context) ([]accounts.Account, error) {
-	return api.db.GetAccounts()
+	accounts, err := api.db.GetAccounts()
+	if err != nil {
+		return nil, err
+	}
+
+	for i := range accounts {
+		account := &accounts[i]
+		if account.Wallet && account.DerivedFrom == "" {
+			address, err := api.db.GetWalletRoodAddress()
```

Comment:
```suggestion
			address, err := api.db.GetWalletRoodAddress()
```

typo?  `Rood` => `Root`

Reply:
Will rename function declaration